src/app/locales/ja/settings.yml

```diff
@@ -0,0 +1,567 @@
+menu:
+    header: 設定
+    main: メイン
+    gui: GUI
```

Comment:
I noticed that GUI wasn't translated and thus stands out in the settings main menu due to this.

I've checked deepl.com for a translation, but "graphical user interface" in Japanese seems way to long for a translation. Are there any alternatives so that it doesn't stand out that much?

DeepL says that "ユーアイ" can be used for "UI", but Google-translate translates it to "you eye", which means it's just a transscription and not a translation.

Btw, I'm aware of all the other borrowed English terms. Those are fine though, because they're not part of any menu items.

Reply:
"ユーアイ" looks strange in Japanese, and "グラフィカルユーザーインターフェース" only means GUI after all, so I translated it as is. I don't think there is a problem because many Japanese know GUI. 
You could use "インターフェース" (interface) or "レイアウト" (layout), but should I do? I think the former is more fitting, but if you would like to reduce the number of characters, I think "レイアウト" is better.

Reply:
Additionally, Japanese fonts often use the alphabet as it is in abbreviations and other ways, so Japanese fonts contain the alphabet. Sometimes we use full-width alphabets for layout (ＧＵＩ etc), but that's not common these days (used to be common in older Japanese software).

Reply:
Ok, thanks for the info. Let's leave it as it is.

If there aren't any further issues or questions, I'll update/fix this PR later today and then merge it.